src/dda/secrets/ssm.py

```diff
@@ -0,0 +1,6 @@
+def fetch_secret_ssm(name: str) -> str:
```

Comment:
```suggestion
def fetch_secret(name: str) -> str:
```
As it's in the `ssm` namespace I guess it's ok. Just to align on the naming from the `vault` module